### PR TITLE
Created feature branch for my Mobile Web Project

### PR DESCRIPTION
**Pull request by HAMMAS-SALEEM**
### This is my second repository in Microverse Program and my first in Mobile Web.

**Taks which i did :memo:**

:heavy_check_mark: Added an markup (HTML)
:heavy_check_mark: Added an stylesheet (CSS)
:heavy_check_mark: Created a Homepage for my portfolio
:heavy_check_mark: Added a .gitignore file
:heavy_check_mark: Setup HTML & CSS Linters
:heavy_check_mark: Documented work in a professional way

###IMPORTANT NOTE FOR REVIEWERS###

NOTE: This is the second time I am submitting my repository. You'll not find the first reviews in my pull request because after the first submission I make a big mess to my repository so I made it again with the same name and everything the same. I want Microverse to forgive me for this mistake. By the way, I have added some screenshots at the end of my previous review so you can figure out the changes I've made according to the first review. I've already told Microverse success team about this mistake.
###LINK OF MY OLD REPOSITORY AFTER FIRST SUBMISSION###
https://github.com/HAMMAS-SALEEM/Portfolio2/pull/2

###OLD REPOSITORY SNAPSHOTS AFTER FIRST SUBMISSION###

old-repository-01
old-repository-02
old-repository-03
old-repository-04

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+                    <p class="connect">LET'S CONNECT</p>
+                    <ul class="icon-list">
+                        <li><a href="#" class="fa fa-twitter fa-color" aria-hidden="true"></a></li>
+                        <li><a href="#" class="fa fa-linkedin fa-color" aria-hidden="true"></a></li>
+                        <li><a href="#" class="fa fa-medium fa-color" aria-hidden="true"></a></li>
+                        <li><a href="#" class="fa fa-github fa-color" aria-hidden="true"></a></li>
+                        <li> <a href="#" class="fa fa-angellist fa-color" aria-hidden="true"></a></li>
+                    </ul>
+                </div>
+            </div>
```
Comment (andersonlebon) [inline index.html]: - [ ] good job :clap: :clap: Good job I can see you implemented the changes given by the first reviewer. However, one more last thing to fix. 
- Your `<nav>` tag should be in a `<header>` tag
- Your section should not be in the `<header>` tag
You should end up with this skeleton : 

````
 <header>
       <nav>
            content
       </nav>
 </header>
<section>
       content
</section>
````
KIndly implement these changes and I hope the next one will be approved :+1: :+1:

**Review by andersonlebon — CHANGES_REQUESTED** [review]
Hi @HAMMAS-SALEEM 

Good job so far! I know you have worked hard on making changes requested by the first reviewer but there are some issues that you still need to work on to go to the next project but you are almost there!

almost-there
## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- You  are following GitHub actions :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Your design looks good :100: :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
## Required Changes ♻️

- [ ] Kindly check comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by HAMMAS-SALEEM** [comment]
Sir Thanks for understanding my issue. I wanna say my first reviewer said
that you should have a header as the parent container for your section he
didn't mention this change and you are saying to remove the header for the
section and make it parent for the navbar.

Sir, please guide me in this situation.

**Comment by HAMMAS-SALEEM** [comment]
Now I have done as the second reviewer said so let's hope that the third reviewer will approve my project Inshallah.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <script src="https://use.fontawesome.com/303b1d319c.js"></script>
+    <title>Portfolio</title>
+</head>
+
+<body>
+    <header>
+        <nav>
+            <div class="container0">
+                <a href="#" class='logo'>My Logo</a>
```
Comment (mmsesay) [inline index.html:23]: - [ ] [OPTIONAL] Your site's logo should be out of the nav.

See example:
```
<header>
   <a> //your logo goes here </a>
   <nav> // your hamburger icon goes here </nav>
</header>
```

**Review by mmsesay — APPROVED** [review]
Hi @HAMMAS-SALEEM , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**